sdk/storage/azure-storage-blob-cryptography/CHANGELOG.md

```diff
@@ -3,10 +3,11 @@
 ## 12.27.0-beta.2 (Unreleased)
 
 ### Features Added
+- Added a new `EncryptionVersion.V2_1` that allows  encrypted blobs to be uploaded using a configurable authenticated region length.
```

Comment:
Small nit, I think there are two spaces between `allows` and `encrypted`. Let's remove one of them